src/common/types/core.js

```diff
@@ -269,6 +272,21 @@ type SharedLink = {
     vanity_url?: string,
 };
 
+type SharedLinkUpdateType = {
```

Comment:
This seems like it's been duplicated from the `SharedLink` type with all optional parameters. Should we use `$Shape<SharedLink>`, instead?